sysl2/sysl/cmd_runner.go

```diff
@@ -105,3 +121,73 @@ func EnsureFlagsNonEmpty(cmd *kingpin.CmdClause, excludes ...string) {
 
 	cmd.PreAction(fn)
 }
+
+func (r *cmdRunner) getProjectRoot(fs afero.Fs, logger *logrus.Logger) error {
+	rootIsDefined := r.Root != ""
+
+	modulePath := r.module
+	if rootIsDefined {
+		modulePath = filepath.Join(r.Root, r.module)
+	}
+
+	syslRootPath, err := findRootFromSyslModule(modulePath, fs)
```

Comment:
I'm not sure I see the point of searching for a .sysl/ if the root arg was supplied, just unconditionally warn that the are is no longer required and should be replaced with the folder marker. Doing that would simplify this func

Reply:
It just warns the user that a `.sysl` is found, but if user still wants to set their own root that is still allowed. Would that warning be useful or unnecessary?

Reply:
@anzdaddy

Reply:
This was my suggestion, but I'm not particularly wedded to one approach or another. We'll learn and adapt.